src/components/Surface/Surface.js

```diff
@@ -321,7 +321,11 @@ function Surface() {
   };
 
   useEffect(() => {
-    context.trigger('resize');
+    requestAnimationFrame(() => {
+      // Trigger custom event instead of H5P event to avoid
+      // triggering resize until breakpoints have been applied.
+      context.trigger('resizecontenttype');
```

Comment:
We could even do `context.resize()` and avoid the custom event. Not sure what is more readable.

Reply:
Yes, it works

Reply:
Edit: no it didn't work as it was not possible to drag arguments

Reply:
That's weird. Does it work to drag arguments with the current changes?

Reply:
Yes